10/src/javascripts/app.js

```diff
@@ -0,0 +1,25 @@
+let wd = require('./wd.js');
+
+window.onload = function () {
+    wd.createRoot('app')
+        .routes([
+            { url: '/', templateUrl: 'routes/home.html' },
+            { url: '/sign-in', templateUrl: 'routes/sign-in.html' }
+        ])
+        .component("RouterLink", {
+            template: "<a href=\"\"><\/a>",
+            beforeMount: function (app, element, componentData) {
+                //console.log("beforeMount", app, element, componentData);
+                let replacement = document.createElement("a");
+                replacement.href = element.getAttribute("href");
+                replacement.innerHTML = element.innerHTML;
+                replacement.addEventListener("click", function (e) {
```

Comment:
It's not the part of the task and you don't need to fix it, but you should know about potential problem that you can face with. You are using here an anonymous function which will not allow you to remove this listener from the element. 
Next time use named function, to be able to 
```javascript
replacement.addEventListener("click", listenerFn);
// somewhere after removing the replacement element from the DOM
replacement.removeEventListener("click", listenerFn);
```